### ISSUE TITLE
Notify users in private when they are added to a channel ACL

### ISSUE THREAD

**Issue by ghost** [shalture/shalture#4]
It might be nice to have a user flag to receive a private notice (or message if you have the `PRIVMSG` flag) when you're added to a channel access list. For example:

```
-ChanServ(ChanServ@services.)- alice has added you to the access list for #wonderland with flags +Aeiortv.
```

This should probably include adding or removing individual flags and removing channel access completely as well.

**Questions**

Feel free to add more below.
- Should the user be told who added them to the access list? (Maybe that should be dependent on the state of the `PRIVATE` flag.)
- Should the user be notified when they are akicked from a channel?

**Comment by GLolol** [comment]
I agree with using `VERBOSE` for this. However, if someone is already on the channel that they got access added, I think the private message should be skipped to prevent duplicate notices.

If `VERBOSE` is set to `OPS`, should the target user only get a notice if they had `+Oo` added onto them? Also, What if the target user is offline? Could MemoServ be configured to send these notices (with a user flag to toggle this behavior)?
